### ISSUE TITLE
Can't add new password

### ISSUE THREAD

**Issue by MaximilianKindshofer** [jakeswenson/BitBetter#120]
I just upgraded to the newest Version (1.43.0) and now I can't add passwords due to Internal Server Errors

**Comment by captainhook** [comment]
I don't believe this is BitBetter related but you'll need to provide some useful information for anyone to assist you with this issue - logs would be a good start.

**Comment by JeffAlyanak** [comment]
@MaximilianKindshofer For what it's worth, many former contributors to BitBetter, myself included, have switched to Vaultwarden.

**Comment by MaximilianKindshofer** [comment]
Hi Jeff, I switched yesterday. Works really good. Just a little wistful. Always liked BitBetter. It is a great project
